app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/service/NoteService.java

```diff
@@ -0,0 +1,12 @@
+package br.org.cesar.discordtime.stickysessions.data.remote.service;
+
+import br.org.cesar.discordtime.stickysessions.data.remote.model.NoteRemote;
+import br.org.cesar.discordtime.stickysessions.domain.model.Note;
+import io.reactivex.Single;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
+
+public interface NoteService {
```

Comment:
Please add a test for this interface (using `RemoteServiceFactory`) to make sure JSON expected from server can be mapped to our NoteRemote definition. Please refer to `SessionServiceTest` class.